canonical-websites/www.ubuntu.com
# 1675: Fix styling issues with bespoke heading + icon markup/scss

## Done

- Split out custom `.p-heading-icon` modifiers into their own file, and updated to work with new Vanilla spacing
- Updated markup where bespoke heading icon classes were used to remove unnecessary utilities
- Updated `.p-heading--insights__title` to work with new Vanilla spacing

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check that the styling of the Insights strip looks good
- Go to http://0.0.0.0:8001/financial-services
- Check that the muted heading icons look good (e.g. Webinar, Article etc)
- Check that the small heading icons look good (e.g. "Products for the finance industry" strip)

## Issue / Card

Fixes vanilla-framework/vanilla-framework#1675
